src/Public/Responses.ps1

```diff
@@ -785,6 +782,82 @@ function Write-PodeXmlResponse {
     Write-PodeTextResponse -Value $Value -ContentType 'text/xml' -StatusCode $StatusCode
 }
 
+
+
+
+<#
+.SYNOPSIS
+Writes YAML data to the Response.
+
+.DESCRIPTION
+Writes YAML data to the Response, setting the content type accordingly.
+
+.PARAMETER Value
+A String, PSObject, or HashTable value. For non-string values, they will be converted to YAML.
+
+.PARAMETER Path
+The path to a YAML file.
+
+.PARAMETER Depth
+The Depth to generate the YAML document - the larger this value the worse performance gets.
+
+.PARAMETER StatusCode
+The status code to set against the response.
+
+.EXAMPLE
+Write-PodeYamlResponse -Value '{"name": "Rick"}'
+
+.EXAMPLE
+Write-PodeYamlResponse -Value @{ Name = 'Rick' } -StatusCode 201
+
+.EXAMPLE
+Write-PodeYamlResponse -Path 'E:/Files/Names.json'
+#>
+function Write-PodeYamlResponse {
```

Comment:
Does this not need a final check on the value if it's empty - like JSON's does? Ie:
```powershell
if ([string]::IsNullOrWhiteSpace($Value)) {
    $Value = '[]'
}
```